src/ui/Templates/HtmlTemplate.ts

```diff
@@ -19,7 +19,10 @@ export class HtmlTemplate extends Template {
 
   toHtmlElement(): HTMLElement {
     let script = this.templateFromAScriptTag.toHtmlElement();
-    script.setAttribute('type', _.first(HtmlTemplate.mimeTypes));
+    // We don't set the type attribute for 2 reasons:
```

Comment:
I will add an UT, it's a great idea.

It leaves a warning in the console and "ignore the method" each time a template is used/created. So we really have to be great when the type is not set and correctly default to HTMLTemplates.